.claude/skills/roughcut/templates/apply_recipe.py

```
@@ -184,6 +236,169 @@ def _apply_speed_ramps(self, item, clip, idx):
+        try:
+            raw = item.GetFusionCompCount()
+            count = int(raw) if raw is not None else 0
+        except (TypeError, ValueError):
+            return out
+        for i in range(max(count, 0)):
+            try:
+                c = item.GetFusionComp(i)
+                if c:
+                    out.append(c)
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)